spotistat.py

```diff
@@ -262,6 +283,10 @@ def main(stream_file_list):
 
 
 if __name__ == "__main__":
-    if len(sys.argv) >= 2:
-        print(f'sys: {sys.argv[1:]}')
-        main(sys.argv[1:])
+    all_files = [pos_json for pos_json in os.listdir(os.getcwd())
```

Comment:
Nice approach, I suppose years ago this did not cross my mind (newbie experience), although I remember having clear problems with using multiple data files.